src/app/utils/UserUtil.js

```diff
@@ -21,11 +21,13 @@ export const setStore = (theStore) => {
  * @returns {boolean}
  */
 export const currentUser = () => {
-    const state = store.getState();
-    if(state.user) {
-        const current = state.user.getIn(['current']);
-        if(current) {
-            return current;
+    if (store) {
```

Comment:
Shouldn't it be a hard requirement to call setStore before these methods? ie should it not be strict and throw an exception?

Reply:
That was my initial thought. 

@bnchdrff changed my mind about that. Maybe a console statement would be appropriate.

Reply:
did you look into delaying scroll until content & images are done loading?

Reply:
did you look into delaying scroll until content & images are done loading?